README.md

```diff
@@ -7,6 +7,8 @@ This is the source repository for the Baïkal CalDAV and CardDAV server.
 
 Head to [sabre.io/baikal][2] for information about installation, upgrading and troubleshooting.
 
+A German tutorial about installation of Baïkal, using it together with Thunderbird, Android and DavX5, sharing calendars, database maintenance and security can be found [here][6].  
```

Comment:
I just had time to read the tutorial in detail. It seems to be focused a lot on the Raspberry Pi, so this should be mentioned here.

Some remarks about the content of the tutorial:
- The install section mentions that calendar sharing is not supported. I would change that paragraph to be up-to-date instead of writing something outdated with an "update". For readers, it does not really matter that there was an old version without support.
- I am confused about the paragraph about the Influxdb port. I would move it to the end of the web server configuration section. Before that, you did not tell that you plan to use port 9999
- The thing that blocks the specific folder in the nginx config should also block the config folder. This is a security risk otherwise. You root seems to be the html folder, though, so specific and config are not accessible anyway. So depending on what you want to do, please either add the config folder or remove that section of the config file because it does not apply in your case.
- Creating the nginx config in the Baikal root folder is pretty strange. It is usually created in the sites-available folder of nginx. The problem I see with having it in the Baikal folder is that the file might be overwritten when updating the Baikal folder.
- I would remove the password security calculation. This assumes that an attacker wants to find your exact password. The thing is, md5 allows to calculate "backwards" to get a password (usually not the one you chose) that has the same hash. So independently of the password, "only" 2^128 tries are needed to get in. Not sure how feasible this is. This would need some more research but the (maybe wrong) claim can be avoided by removing that paragraph :)
- I would remove the sentence that the email address is not important. It is when sharing, so nobody should be tempted to enter something wrong.
- You mix install via composer (experts only) and via zip file (without composer, for most users). This could be confusing.
- The user login section is pretty unexpected. Normal users (no sharing) do not need to log in. This should have a paragraph about when this is needed.
- clearing addressbookchanges is not advisable as is breaks sync for clients that have not been connected for a long time. I do not think that this needs a significant amount of storage.